frontend/lib/src/components/widgets/TextArea/TextArea.tsx

```diff
@@ -121,14 +133,25 @@ const TextArea: FC<Props> = ({
     onFormCleared,
   })
 
+  useEffect(() => {
```

Comment:
**suggestion (non-blocking):** Would be great if we encapsulated this into a shared hook since TextInput also needed this same fix. I'd imagine most of the input-style elements need this

Reply:
Yep totally agree! I will follow-up with a PR after the merge to not block the cherry-pick 🙂

Reply:
Followed up with this PR: https://github.com/streamlit/streamlit/pull/9835